mungegithub/Makefile

```diff
@@ -69,6 +69,12 @@ secret:
 clean:
 	rm -f mungegithub $(APP)/local.deployment.yaml $(APP)/local.secret.yaml
 
+# pull down current public queue state, and run UI based off that data
+ui-stub:
+	@/bin/bash -c "wget -r -nH -P ./submit-queue/www http://submit-queue.k8s.io/{prs,github-e2e-queue,history,sq-stats,stats,users,health,google-internal-ci,priority-info,merge-info,health}; \
+	pushd ./submit-queue/www; \
+	python -m SimpleHTTPServer;"
```

Comment:
TIL

On Wed, Jun 15, 2016 at 10:08 AM, goltermann <notifications@github.com>
wrote:

> In mungegithub/Makefile
> <https://github.com/kubernetes/contrib/pull/1209#discussion_r67203363>:
>
> > @@ -69,6 +69,12 @@ secret:
> >  clean:
> >  	rm -f mungegithub $(APP)/local.deployment.yaml $(APP)/local.secret.yaml
> >
> > +# pull down current public queue state, and run UI based off that data
> > +ui-stub:
> > +	@/bin/bash -c "wget -r -nH -P ./submit-queue/www http://submit-queue.k8s.io/{prs,github-e2e-queue,history,sq-stats,stats,users,health,google-internal-ci,priority-info,merge-info,health}; \
> > +	pushd ./submit-queue/www; \
> > +	python -m SimpleHTTPServer;"
>
> It's basically
> http://stackoverflow.com/questions/5193048/bin-sh-pushd-not-found. If I
> understand correctly, $(shell ...) uses the default shell, which may not
> have pushd available.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/contrib/pull/1209/files/71d8322ddcc39adcf0769d3f805f02006e7e5538#r67203363>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAngluAg3AZoNCGWRPLD7Qzn3s4e2SSRks5qMDGDgaJpZM4I13qN>
> .
>